editor/components/default-block-appender/test/__snapshots__/index.js.snap

```diff
@@ -35,22 +35,7 @@ exports[`DefaultBlockAppender should append a default block when input focused 1
 </div>
 `;
 
-exports[`DefaultBlockAppender should match snapshot 1`] = `
-<div
-  className="editor-default-block-appender"
->
-  <Connect(WrappedComponent) />
-  <input
-    className="editor-default-block-appender__content"
-    onClick={[MockFunction]}
-    onFocus={[MockFunction]}
-    onKeyDown={[MockFunction]}
-    readOnly={true}
-    type="text"
-    value="Write your story"
-  />
-</div>
-`;
+exports[`DefaultBlockAppender should match snapshot 1`] = `""`;
```

Comment:
Should we update the test to check for value `""` instead of using snapshot after this code got refactored?